src/broker/module.c

```diff
@@ -23,6 +23,7 @@
 #endif
```

Comment:
Commit message for c9096f74abbaa31d852de3e4c88a960d1f1d62f5

This might be the commit that people will find in the history when they ask "why isn't libzmqutil part of libflux"?  So it might be good to mention the FIPS startup penalty and its adverse effect on job throughput in the commit message problem statement, even though the issue reference does provide that background.   Something like:

> Isolate 0MQ functions that depend on libzmq in a non-exported convenience library, to make it possible to minimize the number of components linked with libzmq.   On systems configured for FIPS compliance, the cyrpto libs used by libzmq perform a self-test in the dso init function, which slows down library loading.  When 0MQ functions were part of libflux-core.so, this significantly impaired job throughput.